README.md

```diff
@@ -1,2 +1,14 @@
 # test-infra
-test-infra configurations
+
+test-infra is a CI/CD system aimming at ByteDance's Open Source Project Management.
```

Comment:
Let's change `Bytedance` to `kitex-contrib` now.

Reply:
OK